aws-codeguruprofiler-profilinggroup/sample-template.json

```diff
@@ -2,10 +2,43 @@
   "AWSTemplateFormatVersion": "2010-09-09",
   "Description": "Sample template for the AWS::CodeGuruProfiler::ProfilingGroup resource.",
   "Resources": {
+    "MyRole": {
```

Comment:
Minor/ Personal opinion: Possibly naming this test role to something more ProfilingGroup-related: e.g. ProfilingGroupCFTestRole.

Reply:
Renaming it to ```MyProfilingGroupRole```, thanks.